engines/python/setup/djl_python/properties_manager/properties.py

```diff
@@ -57,6 +57,8 @@ class Properties(BaseModel):
     output_formatter: Optional[Union[str, Callable]] = None
     waiting_steps: Optional[int] = None
     is_mpi: bool = False
+    tgi_compat: Optional[bool] = os.environ.get("OPTION_TGI_COMPAT",
```

Comment:
if you are fetching from here, you could just check whether `tgi_compat` is set right? DJLServing will be able to pick it up and trim the `OPTION_` part of it

Reply:
:) Yeah, right!! Let me remove this

Reply:
Done